### PR TITLE
[Bigtable] Add user agent to requests

### PR DESCRIPTION
**Pull request by evildour**

### CODE REVIEW

File: apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs [hunk 1]
```diff
@@ -72,7 +75,10 @@ public abstract partial class BigtableClient
             var channelOptions = new[]
             {
                 new ChannelOption(ChannelOptions.MaxSendMessageLength, -1),
-                new ChannelOption(ChannelOptions.MaxReceiveMessageLength, -1)
+                new ChannelOption(ChannelOptions.MaxReceiveMessageLength, -1),
+
+                // TODO: Figure out if there's a good way to test this
+                new ChannelOption(ChannelOptions.PrimaryUserAgentString, $"cbt-csharp/v{versionAttribute.InformationalVersion}")
```
Comment (jskeet) [inline apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs]: Cache this string in a single place, perhaps?

Comment (evildour) [reply apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs]: Done

Comment (evildour) [reply apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs]: Oops, forgot to actually run it.

Comment (evildour) [reply apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs]: (I forgot to actually get the version string from the attribute in the cached string, so it used the type name)

File: apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs [hunk 2]
```diff
@@ -29,6 +29,9 @@ namespace Google.Cloud.Bigtable.V2
 {
     public abstract partial class BigtableClient
     {
+        private static string UserAgent { get; } =
+            $"cbt-csharp/v{typeof(BigtableClient).GetTypeInfo().Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>().InformationalVersion}";
```
Comment (jskeet) [inline apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs]: Annoyingly, I'd thought about "What does the ToString() override do?" but had assumed it was okay...

**Pull request merged**